### DISCUSSION TITLE
[Bug] Desktop browser window detected as mobile view

### DISCUSSION THREAD

**Discussion by tunaworks** [VikeLabs/courseup#302] [General]
OS: Windows 11
Browser: Chrome 96.0.4664.93

Issue: home page (Possibly index.tsx?)  incorrectly detects desktop browser windows with <1020px width (innerwidth?) as being in mobile mode.  

Screenshot:
![image](https://user-images.githubusercontent.com/19608747/145880989-ece02b25-5354-46df-9613-e4eb1b5b3d56.png)

**Comment by keithradford** [comment]
Hi there @tunaworks ! Thanks for reporting. This isn't actually an unexpected behavior since there's no actual detection to if you're on desktop or not. Since we don't have responsive UI on our major pages (WIP, I swear 😅) we automatically show this modal on any small screen.
